### PR TITLE
hotfix/1.0.2

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: webapp/src/main/webapp/src/assets/public/landing.html [hunk 1]
```diff
@@ -40,11 +40,12 @@
   <div id="about" class="content">
     <h3>Learn More About Smarter Balanced Reporting:</h3>
     <ul>
-      <!--TODO: Remove inline style -->
-      <li class="" style="cursor:default;">
-        <div class="annotatedReports asset"></div>
-        <h2>Interpretive Guide</h2>
-        <p>Learn how to interpret scores in this reporting tool <b>- Available September 15th</b></p>
+      <li class="">
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/public/landing.html]: what does this ghost css class do?

File: webapp/src/main/webapp/src/styles.less [hunk 2]
```diff
@@ -167,4 +167,6 @@ footer {
   }
 }
 
-
+.cursor-pointer {
```
Comment (esotrope) [inline webapp/src/main/webapp/src/styles.less]: I don't think we'll need this now that the interpretive guide href has a value

File: webapp/src/main/webapp/src/assets/public/landing.html (hunk 1 above)
Comment (bgsavage) [reply webapp/src/main/webapp/src/assets/public/landing.html]: A whole lot of nothing.

File: webapp/src/main/webapp/src/styles.less (hunk 2 above)
Comment (esotrope) [reply webapp/src/main/webapp/src/styles.less]: Don't see the comment anymore but I agree this should be removed on develop and isn't critical for this hotfix.

**Pull request merged**